### ISSUE TITLE
Doesn't work with lambdas

### ISSUE THREAD

**Issue by keathley** [keathley/pointfree.io#2]
This snippet produces no output

```
(\f -> f . sum . map (f.f))
f x = \y -> y x
```

**Comment by tfausak** [comment]
The `->` token is tripping up the command-line option parser. You need to let `pointfree` know that it shouldn't expect any more options by using `--`. 

```
>>> readProcess "pointfree" ["\\", "x", "->", "x"] ""
pointfree: user error (unrecognized option `->'
Usage: pointfree [OPTION...] query
  -v  --verbose  verbose results
)
*** Exception: readProcess: pointfree "\\" "x" "->" "x" (exit 1): failed

>>> readProcess "pointfree" ["--", "\\", "x", "->", "x"] ""
"id\n"
```

Alternatively you can use the pointfree library (bmillwood/pointfree#9) available in `pointfree-1.1`. Then you won't have to deal with shell arguments at all.

**Comment by keathley** [comment]
Thanks for the point about the library version of pointfree!  I hadn't seen that.

Fixed in #3
